options.go

```diff
@@ -15,87 +15,69 @@
 package greptime
 
 import (
+	"crypto/tls"
 	"time"
 
 	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
 	"google.golang.org/grpc/credentials/insecure"
 	"google.golang.org/grpc/keepalive"
 )
 
 var (
-	uaOpt = grpc.WithUserAgent("greptimedb-ingester-go/" + Version)
-
-	// TODO(yuanbohan): SecurityOptions
+	uaOpt       = grpc.WithUserAgent("greptimedb-ingester-go/" + version)
+	secureOpt   = grpc.WithTransportCredentials(credentials.NewTLS(&tls.Config{InsecureSkipVerify: false}))
 	insecureOpt = grpc.WithTransportCredentials(insecure.NewCredentials())
-
-	defaultKeepaliveInterval = 30 * time.Second
-	defaultKeepaliveTimeout  = 5 * time.Second
 )
 
-type Options struct {
-	keepalive *KeepaliveOption
+type options struct {
+	secure    *grpc.DialOption
+	keepalive *grpc.DialOption
```

Comment:
suggestion: If we define a `TLSOptions`, maybe `keepalive` is not a proper field. It can be grouped by other struct.

Reply:
yes. `secure` is not a good naming pattern, which has eluded many details. 

`options` will keep the name, and `secure` will be replaced by `tlsOptions`